src/game/tools.py

```diff
@@ -3,6 +3,20 @@
 EMPTY_POSITION = '.'
 
 
+class Players:
```

Comment:
It's very strange to have a "Players" class in "tools.py". Moreover, it seems that you use Players class as a property container, i'm not sure that it's the best way to achieve this.

Reply:
It was also destined to store players scores (other story)

Reply:
In general, all this informations are stored in only one object (a "game" state) to ensure data integrity and synchronicity.